diff --git a/launch/ydlidar_launch.py b/launch/ydlidar_launch.py
index 579c4f4..9f7935e 100755
--- a/launch/ydlidar_launch.py
+++ b/launch/ydlidar_launch.py
@@ -43,14 +43,15 @@ def generate_launch_description():
                                 parameters=[parameter_file],
                                 namespace='/',
                                 )
-    tf2_node = Node(package='tf2_ros',
-                    executable='static_transform_publisher',
-                    name='static_tf_pub_laser',
-                    arguments=['0', '0', '0.02','0', '0', '0', '1','base_link','laser_frame'],
-                    )
+    # tf2_node = Node(package='tf2_ros',
+    #                 executable='static_transform_publisher',
+    #                 name='static_tf_pub_laser',
+    #                 arguments=['0', '0', '0.02','0', '0', '0', '1','base_link','laser_frame'],
+    #                 )   # /scan 토픽만 발행하도록 수정
 
     return LaunchDescription([
         params_declare,
         driver_node,
-        tf2_node,
+        # tf2_node,
     ])
+
